tvb_library/tvb/basic/logger/library_logger.conf

```diff
@@ -5,64 +5,64 @@
 keys=root, tvb, tvb_basic_datatypes, tvb_basic_config, tvb_simulator, tvb_simulator_monitors, tvb_traits, numba, matplotlib
 
 [handlers]
-keys=consoleHandler,fileHandler
+keys=consoleHandler, fileHandler, elasticSearchHandler
```

Comment:
The elastic push of logs is expected from Tvb-framework (web interface), also we should be able to switch elastic handler on/off from a config param. The curent code does not handle the 2 aspecte above, we should speak/think how to best meet these